scripts/rewrite/resources/post-to-canny.py

```
@@ -0,0 +1,245 @@
+            data = json.loads(resp.read().decode())
+    except Exception as e:
+        die(f"Canny posts/create failed: {e}")
+
+    # The API returns the created post with `id` + `url`. The URL on the
+    # response can be user-facing (canny.io/p/...) or an admin URL. We'll
+    # attach whichever is present.
+    return data
+
+def gh_pr_comment(pr: int, body: str) -> None:
+    try:
+        subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)